Documentation/network/concepts/ipam/eni.rst

```diff
@@ -308,15 +308,28 @@ is performed at the interval based scan.
 When determining whether a node has a deficit in IP addresses, the following
 calculation is performed:
 
+``availableIPs``: all IPs on ENI contain used IPs and not used IPs.
```

Comment:
Why can't we not use `len(spec.ipam.available)` instead?

Reply:
The `spec.ipam` field does not contain the `available` attribute. https://github.com/wu0407/cilium/blob/8c77e5b059a80d7a1bcf9a213437ec98cc8ae4d3/pkg/ipam/types/types.go#L121

Reply:
Ah true, this should be `spec.ipam.pool`, right?

Reply:
yeah